app/models/warehouse/case_report.rb

```diff
@@ -281,10 +281,15 @@ def process_class_related_process(kase, case_report)
         end
       end
 
+      def total_cached_num_pages(kase)
+        kase.data_request_areas&.decorate&.sum(&:cached_num_pages) || 0
```

Comment:
It doesn't seem right that this would require a decorator which suggests to me that the `cached_num_pages` method should be available on the object.